### PR TITLE
Add docs for installing in Linux

### PR DESCRIPTION
**Pull request by jorgebay**
Installation and basic usage instructions for linux.

### CODE REVIEW

**Comment by tanberry** [comment]
@jorgebay will you be merging this PR, or updating it again, or...? If I have comments/edits, do you want them against this PR?

**Comment by jorgebay** [comment]
@tanberry yes, please send comments/suggestions over this PR.

File: docs/LINUX.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+# Usage
+
+You can start sending logs by setting the [ingestion key][ingestion-key]:
+
+```shell script
+logdna-agent -k <YOUR INGESTION KEY>
+```
+
+The agent expose short argument abbreviations for commonly used options (`-k`, `-t`, `-d` and `-c`).
+
```
Comment (tanberry) [inline docs/LINUX.md:38]: Suggest we say "The LogDNA Agent exposes short..." (need the "s" in "exposes" for sure.)

File: docs/LINUX.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+FLAGS:
+    -h, --help       Prints help information
+    -V, --version    Prints version information
+
+OPTIONS:
+    -c, --config <config>
+            The config filename [env: LOGDNA_CONFIG_FILE=]  [default: /etc/logdna/config.yaml]
+
+        --db-path <db-path>
+            The directory the agent will store it's state database. Note that the agent must have write access to the
```
Comment (tanberry) [inline docs/LINUX.md:59]: "The directory in which the agent will store its state database." (No apostrophe in "its".)

File: docs/LINUX.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+            directory and be a persistent volume. Defaults to "/var/lib/logdna-agent/" [env: LOGDNA_DB_PATH=]
+        --endpoint-path <endpoint-path>
+            The endpoint to forward logs to. Defaults to "/logs/agent" [env: LOGDNA_ENDPOINT=]
+        --exclude-regex <exclusion-regex>...
+            List of regex patterns to exclude files from monitoring [env: LOGDNA_EXCLUSION_REGEX_RULES=]
+        --exclude <exclusion-rules>...
+            List of glob patterns to exclude files from monitoring, to add to the default [env: LOGDNA_EXCLUSION_RULES=]
```
Comment (tanberry) [inline docs/LINUX.md:66]: "... to add to the default ----what exactly---? Does it mean "... to add to the default set of exclusion rules? Or to the default value... we need to qualify the term "default". ;-)

File: docs/LINUX.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        --gzip-level <gzip-level>
+            If compression is enabled, this is the gzip compression level to use. Defaults to 2 [env:
+            LOGDNA_GZIP_LEVEL=]
+        --host <host>
+            The host to forward logs to. Defaults to "logs.logdna.com" [env: LOGDNA_HOST=]
+        --include-regex <inclusion-regex>...
+            List of regex patterns to include files from monitoring [env: LOGDNA_INCLUSION_REGEX_RULES=]
+        --include <inclusion-rules>...
+            List of glob patterns to includes files for monitoring, to add to the default (*.log) [env:
```
Comment (tanberry) [inline docs/LINUX.md:75]: Similar comment as above... the default what? Does it mean the default list of files to include (which by default include any files with the extension .log?

Also, I am not sure what the first part of sentence means: "List of glob patterns to includes files for monitoring..." Does it mean include any files that contain this GLOB?

File: docs/LINUX.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+        --journald-paths <journald-paths>...
+            List of paths (directories or files) of journald paths to monitor, for example: /var/log/journal or
+            /run/systemd/journal [env: LOGDNA_JOURNALD_PATHS=]
+    -k, --key <key>
+            The ingestion key associated with your LogDNA account [env: LOGDNA_INGESTION_KEY=]
+        --line-exclusion <line-exclusion>...
+            List of regex patterns to exclude log lines. When set, the Agent will NOT send log lines that match any of
+            these patterns [env: LOGDNA_LINE_EXCLUSION_REGEX=]
+        --line-inclusion <line-inclusion>...
+            List of regex patterns to include log lines. When set, the Agent will ONLY send log lines that match any of
```
Comment (tanberry) [inline docs/LINUX.md:88]: It's a tiny nit, but best to say "...the Agent will send ONLY log lines that match..." (It's typical to place the word "only" to be after the verb.)

File: docs/LINUX.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+            these patterns [env: LOGDNA_LINE_INCLUSION_REGEX=]
+        --line-redact <line-redact>...
+            List of regex patterns used to mask matching sensitive information before sending it the log line [env:
```
Comment (tanberry) [inline docs/LINUX.md:91]: Before sending it **in** the log line...?? 

Maybe "List of regex patterns used to mask matching sensitive information (such as PII) before sending it in the log line..."

File: docs/LINUX.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+            LOGDNA_REDACT_REGEX=]
+    -d, --logdir <log-dirs>...
+            Adds log directories to scan, in addition to the default (/var/log) [env: LOGDNA_LOG_DIRS=]
+        --log-k8s-events <log-k8s-events>
+            Whether the agent should log Kubernetes resource events. This setting only affects tracking and logging
```
Comment (tanberry) [inline docs/LINUX.md:96]: "Configures whether the ..."or "Determines whether the..."

**Review by tanberry — APPROVED** [review]

File: docs/LINUX.md (hunk 4 above)
Comment (jorgebay) [reply docs/LINUX.md:75]: yes, its a little confusing... the line comes originally from https://github.com/logdna/logdna-agent-v2#options

Comment (jorgebay) [reply docs/LINUX.md:75]: It means that it will only include files matching this rule.

**Comment by jorgebay** [comment]
I've addressed the suggestions, I'll merge it as it's a good starting point for the beta.

**Pull request merged**